aws/provider.go

```diff
@@ -375,6 +375,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_config_delivery_channel":                             resourceAwsConfigDeliveryChannel(),
 			"aws_config_organization_custom_rule":                     resourceAwsConfigOrganizationCustomRule(),
 			"aws_config_organization_managed_rule":                    resourceAwsConfigOrganizationManagedRule(),
+			"aws_config_remediation_configuration":                    resourceAwsConfigRemediationConfiguration(),
```

Comment:
This new resource is missing documentation as noted in the [Contributing Guide section for new resources](https://github.com/terraform-providers/terraform-provider-aws/blob/master/.github/CONTRIBUTING.md#new-resource):

- A new resource documentation page at `website/docs/r/config_remediation_configuration.html.markdown`
- A link to the new resource documentation page in `website/aws.erb`

Reply:
I thought I was missing something obvious ^^"

Reply:
I just pushed my first draft of the documentation, please feel free to give me feedback about it.